index.html

```diff
@@ -17,14 +17,21 @@
             <a href="#"><h2>Bilal</h2></a>
         </div>
         <nav class="Menuicon">
-            <span class="material-symbols-outlined">
+            <span id="navicon" class="material-symbols-outlined">
                 menu
                 </span>
         </nav>
-        <ul class="menu">
-            <li><a class="menu_item" href="#portfolio">portfolio</a></li>
-            <li><a class="menu_item" href="#about">About</a></li>
-            <li><a class="menu_item" href="#contact">Contact</a></li>
+        <ul id="menu" class="menu">
+            <a href="#" id="cross" class="Menuiconcross">
+                <span
+                    id="crossIcon"
+                    style="color: white"
+                    class="material-symbols-outlined">
+                        close
+				</span></a>
+            <li id="work_m" class="lil"><a class="menu_item" href="#work">portfolio</a></li>
```

Comment:
- [ ] Great job adding the link to your portfolio, but you're missing an `s` in the spelling of works here as the id you have on your portfolio section which you're trying to get to is spelled `works` . please kindly add an `s` in front of work